source/lib/cache.js

```diff
@@ -269,15 +269,17 @@ export async function fetchWeeklyMovie(
 ): Promise<MaybeError<WeeklyMovie>> {
 	const cachedValue = await getWeeklyMovie()
 
-	if (!isOnline) {
-		if (cachedValue.isCached && cachedValue.value) {
-			return {error: false, data: cachedValue.value}
+	if (process.env.NODE_ENV !== 'development') {
```

Comment:
Invert this if condition; if it's "=== dev", the mini diet can remove it entirely for the production builds.

Reply:
Done